src/uproot_browser/plot.py

```diff
@@ -15,6 +15,10 @@
 import uproot
 
 
+class EmptyTreeError(ValueError):
+    pass
```

Comment:
@henryiii Thoughts on factoring this out into an `exceptions` module? Or do you want there to be more than 1 before doing this sort of thing?

Reply:
Okay either way. I'd usually have an exceptions module, but it was easier to do this for one exception in the PR & it's not really a library.

Reply:
I'm trying to add an `exceptions.py` but `mypy` doesn't like the way that I'm importing it. If that's the only thing holding this PR back from merge maybe just move ahead with it and I can PR it later once I figure out the problem.

Reply:
How are you importing it?

Reply:
```diff
$ git diff --cached
diff --git a/src/uproot_browser/exceptions.py b/src/uproot_browser/exceptions.py
new file mode 100644
index 0000000..f88330e
--- /dev/null
+++ b/src/uproot_browser/exceptions.py
@@ -0,0 +1,4 @@
+"""Custom exceptions for uproot-browser"""
+
+class EmptyTreeError(ValueError):
+    pass
diff --git a/src/uproot_browser/plot.py b/src/uproot_browser/plot.py
index 3b2b6a8..53c2005 100644
--- a/src/uproot_browser/plot.py
+++ b/src/uproot_browser/plot.py
@@ -14,9 +14,7 @@ import numpy as np
 import plotext as plt
 import uproot
 
-
-class EmptyTreeError(ValueError):
-    pass
+from uproot_browser.exceptions import EmptyTreeError
```

```console
$ pre-commit run mypy --all-files
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

src/uproot_browser/plot_view.py:57: error: Module has no attribute "EmptyTreeError"  [attr-defined]
Found 1 error in 1 file (checked 12 source files)
```

Reply:
You need to either import it from `uproot_browser.exceptions ` inside `plot_view.py`, or you need to re-export it from `plot.py` using `__all__`.

Reply:
Ah but now that fails pylint. :/